### DISCUSSION TITLE
CVE-2022-40684

### DISCUSSION THREAD

**Discussion by ghost** [projectdiscovery/nuclei-templates#5690] [General]
Hi 
I would like ask about individual update new CVE .May I individual put new CVE for Nuclei to nuclei-templates/cves/2022 .This will be correct and Nuclei will be use it for scan.Or Nuclei will ignore it.???

**Comment by ehsandeep** [comment]
@kolor33 can you share what update/changes you are looking to make in this CVE?

**Comment by ghost** [comment]
My question is only if I got some private CVE templates .Where is correct place in Nuclei structure put this file .Where ???

**Reply by ehsandeep** [reply]
@kolor33 if it's for the CVE for which the template does not exist, you can follow the current directory structure, i.e `cves/YEAR/CVE_ID`, to add in the public template project.
